sonic_package_manager/manager.py

```diff
@@ -557,11 +568,23 @@ def upgrade_from_source(self,
                     self._get_installed_packages_and(old_package))
                 )
 
-                if self.feature_registry.is_feature_enabled(new_feature):
+                if feature_enabled:
```

Comment:
Here you are using "feature_enabled" obtained for old package. Is that intentional ?
If yes, please add some comments, explaining the use across old & new

Reply:
Added a comment